config/kubernetes/org.yaml

```diff
@@ -2265,7 +2268,9 @@ teams:
       metrics: admin
       mount-utils: admin
       node-api: admin
-      noderesourcetopology-api: admin
+      # TODO: noderesourcetopology-api is part of k-sigs org
+      # noderesourcetopology-api: admin
```

Comment:
The noderesourcetopology-api repo was removed as a staging repo. We should be good to remove this repo assignment and the comment.

Ref: https://github.com/kubernetes/org/issues/4224#issuecomment-1618750306